### PR TITLE
Prepare for the first release

### PR DESCRIPTION
**Pull request by zoetrope**

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+          BRANCH=$(echo ${{ env.tag }} | cut -d "." -f 1-2)
+          docker tag quay.io/cybozu/pod-security-admission:latest quay.io/cybozu/pod-security-admission:$BRANCH
+          docker push quay.io/cybozu/pod-security-admission:$BRANCH
+  release:
+    name: Release on GitHub
+    needs: image
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - run: make build/install.yaml
```
Comment (yokaze) [inline .github/workflows/release.yaml:34]:
```suggestion
      - run: make setup
      - run: make build/install.yaml
```

Possibly missing `make setup`.

```
/path/to/pod-security-admission/bin/kustomize build ./config/default > build/install.yaml
/bin/bash: /path/to/pod-security-admission/bin/kustomize: No such file or directory
make: *** [Makefile:77: build/install.yaml] Error 127
```

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      - name: Create release
+        id: create_release
+        uses: actions/create-release@v1
```
Comment (yokaze) [inline .github/workflows/release.yaml]: `actions/create-release` is no more maintained, so its use should stand on an explicit decision.

File: .github/workflows/release.yaml (hunk 1 above)
Comment (yokaze) [reply .github/workflows/release.yaml:34]: `kustomize:` should be `$(KUSTOMIZE):`.

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -25,31 +26,51 @@ spec:
...
+              path: /readyz
+              port: 8081
+            initialDelaySeconds: 5
+            periodSeconds: 10
+          lifecycle:
+            preStop:
+              exec:
+                command: [ "sleep", "5" ]
+          resources:
+            limits:
```
Comment (yokaze) [inline config/manager/manager.yaml]: We need to remove `limits`.

**Review by yokaze — CHANGES_REQUESTED** [review]

**Comment by zoetrope** [comment]
Pre-release was successful!
https://github.com/cybozu-go/pod-security-admission/releases/tag/v0.0.1-alpha.0
https://quay.io/repository/cybozu/pod-security-admission?tab=tags

**Review by yokaze — APPROVED** [review]
LGTM

**Pull request merged**